Git/syntax_test_git_rebase

```diff
@@ -0,0 +1,38 @@
+# SYNTAX TEST "Packages/Git/rebase.sublime-syntax"
+# <- source.gitrebase comment.line punctuation.definition.comment
+pick d284bb2 Initial commit
+# <- meta.commit storage.type.edit
+#^^^ meta.commit storage.type.edit
+#   ^ meta.commit -storage.type.commit
+#    ^^^^^^^ meta.commit constant.numeric.hex.hash
+#            ^^^^^^^^^^^^^^ meta.commit text.plain.commit-msg
+pick 6746220 Second commit
+# <- meta.commit storage.type.edit
+#^^^ meta.commit storage.type.edit
+#   ^ meta.commit -storage.type.commit
+#    ^^^^^^^ meta.commit constant.numeric.hex.hash
+#            ^^^^^^^^^^^^^ meta.commit text.plain.commit-msg
+
+# Rebase 9e73d21..6746220 onto 9e73d21 (2 commands)
```

Comment:
Maybe scope the two dots as punctuation.separator?

Reply:
I think in a manner of speaking the `..` is a range, not a separator. RegExp uses `constant.other.range` for character ranges, but maybe `punctuation` is better here.

Also, there's a `...` in Git (although it should never appear here). Whoever decides on the scope might want to take that into account.

Reply:
`keyword.operator` maybe?